src/Api/Serializer/JecouteNewsGroupsContextBuilder.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Api\Serializer;
+
+use App\Entity\Jecoute\News;
+
+class JecouteNewsGroupsContextBuilder extends AbstractGroupsContextBuilder
```

Comment:
why do we need this class?

Reply:
I think that with the separation of endpoints, we no longer need this context builder